<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@848.v5a_383b_d14921">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@3773.v505e0052522c">
    <parentIds>
      <string>57</string>
    </parentIds>
    <id>58</id>
    <startId>54</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@1261.va_2ff5204f17e">
      <startTime>1693081766638</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1261.va_2ff5204f17e">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 127</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:668)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:614)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:555)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions>
          <wf.a.ErrorAction_-ErrorId>
            <stackTrace/>
            <suppressedExceptions class="empty-list"/>
            <uuid>28b16b19-1324-44ba-882d-0bac34ddf5fe</uuid>
          </wf.a.ErrorAction_-ErrorId>
        </suppressedExceptions>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>